<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkillsDockWidget</class>
 <widget class="QDockWidget" name="SkillsDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skills</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QWidget" name="">
    <layout class="QVBoxLayout" name="toolbarLayout">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>20</y>
      <width>28</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter:</string>
    </property>
    <property name="buddy">
     <cstring>filterLine</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="filterLine">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>20</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="skillsTable">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>46</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>filterLine</tabstop>
  <tabstop>skillsTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
